plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtStoragePoolDef.java

```diff
@@ -75,6 +82,15 @@ public LibvirtStoragePoolDef(PoolType type, String poolName, String uuid, String
         _targetPath = targetPath;
     }
 
+    public LibvirtStoragePoolDef(PoolType type, String poolName, String uuid, String host, String dir, String targetPath, List<String> nfsMountOpts) {
+        this(type, poolName, uuid, host, dir, targetPath);
+        if (CollectionUtils.isNotEmpty(nfsMountOpts)) {
+            for (String nfsMountOpt : nfsMountOpts) {
+                this._nfsMountOpts.put(nfsMountOpt, null);
```

Comment:
if the value is nul in map, can use Set? else keep them as key-values (`vers -> 4.1 ; noconnect -> 4`)

Reply:
Set would be perfect. Thanks.